src/types/intersectionObserver.ts

```
@@ -0,0 +1,93 @@
+ * @interface HTMLElementEventMap
+ * @description Extends global HTMLElementEventMap to include intersection custom event
+ *
+ * @property {CustomEvent<IntersectionEventDetail>} intersection - Custom event dispatched on intersection changes
+ */
+declare global {
+  interface HTMLElementEventMap {
+    intersection: CustomEvent<IntersectionEventDetail>;
+  }
+}
+
+export {};
```

Biome (1.9.4)

[error] This empty export is useless because there's another export or import.

This export makes useless the empty export.

Safe fix: Remove this useless empty export.


(lint/complexity/noUselessEmptyExport)